pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -534,6 +542,8 @@ func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templ
 		} else {
 			require.Len(t, actual.Objects, 14)
 		}
+	case "appstudio":
+		require.Len(t, actual.Objects, 16)
 	case "test":
 		return // Don't care what objects are defined in the test tier
```

Comment:
I also noticed that you don't verify the ServiceAccount anywhere, nor the related RoleBindings.

So, if you don't want to verify any of the objects from AppStudio, you could just return for the `appstudio` case here as well and skip the tier.